### PR TITLE
dev

### PR DESCRIPTION
**Pull request by RMasnyuk**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+const grid = document.querySelector('#grid');
+
```
Comment (ost-ptk) [inline main.js:2]: need a link for gh-page
you copy this solution from - https://next.plnkr.co/edit/xoqlOD7IfVOeriG0L4po?p=preview&preview

File: main.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+grid.addEventListener('click', function (event) {
+    const target = event.target;
+    if (target.nodeName !== 'TH') {
+        return;
+    }
+    sortColumn(event.target.cellIndex, event.target.dataset.type);
+});
+
```
Comment (gestap) [inline main.js]: need a link for gh-page
you copy this solution from - https://next.plnkr.co/edit/xoqlOD7IfVOeriG0L4po?p=preview&preview

**Review by ost-ptk — CHANGES_REQUESTED** [review]

File: main.js (hunk 1 above)
Comment (RMasnyuk) [reply main.js:2]: No, i didn`t really copy this solution. I tried to do this task step-by-step like on the picture below. Yes, i used this solution to solve the problem, but i did`n copy it. You can notice differences.
fragment

Comment (RMasnyuk) [reply main.js:2]: gh-pages : https://rmasnyuk.github.io/sort-table/

**Pull request merged**